chapter_09_commands.asciidoc

```diff
@@ -158,15 +158,12 @@ def handle_event(
             logger.debug('handling event %s with handler %s', event, handler)
             handler(event, uow=uow)
             queue.extend(uow.collect_new_events())
-        except:
+        except Exception:
```

Comment:
for info Ed, whenever we make a change to a listing, i need to go back to the `code` repo and amend some commits, then `git-rebase` all the history from there on, and re-push branches.  it's nontrivial effort, so that's why i leave some todos rather than "just doing it" sometimes.

Reply:
(exception is any listings with `role="skip"`, they're not tested

Reply:
:grimacing: I should have known better, I had the same experience with pycon.  Will go amend this PR, and be careful in the future.